### PR TITLE
Update script to specific noobaa version and add security for catalog source

### PR DESCRIPTION
**Pull request by subhamkrai**
Update script to specific noobaa version instead of master.

Add security context for catalog source until we upgrade opm to file-based catalog

### CODE REVIEW

File: deploy/deploy-with-olm.yaml [hunk 1]
```diff
@@ -14,7 +14,7 @@ metadata:
   namespace: openshift-storage
 spec:
   targetNamespaces:
-  - openshift-storage
+    - openshift-storage
```
Comment (iamniting) [inline deploy/deploy-with-olm.yaml:17]: Can you remove this change?

**Review by iamniting — CHANGES_REQUESTED** [review]

**Comment by iamniting** [comment]
Hey @subhamkrai The ocs-operator-ci is failing can you pls check?

**Comment by subhamkrai** [comment]
> Hey @subhamkrai The ocs-operator-ci is failing can you pls check?

@iamniting running `make gen-latest-deploy-yaml` removing changes in `deploy-with-olm.yaml` any idea how I can keep the changes?

**Comment by iamniting** [comment]
> > Hey @subhamkrai The ocs-operator-ci is failing can you pls check?
> 
> @iamniting running `make gen-latest-deploy-yaml` removing changes in `deploy-with-olm.yaml` any idea how I can keep the changes?

After digging it a bit it looks like you are required to change the spec here <https://github.com/red-hat-storage/ocs-operator/blob/main/pkg/deploy-manager/subscription.go#L130>

**Review by iamniting — APPROVED** [review]
/lgtm

**Pull request merged**